addons/base_vat/models/res_partner.py

```diff
@@ -820,6 +821,10 @@ def check_vat_de(self, vat):
         is_valid_stnr = stdnum.util.get_cc_module("de", "stnr").is_valid
         return is_valid_vat(vat) or is_valid_stnr(vat)
 
+    def check_vat_il(self, vat):
+        check_func = stdnum.util.get_cc_module('il', 'idnr').is_valid
```

Comment:
According to https://github.com/arthurdejong/python-stdnum/blob/master/stdnum/il/hp.py, shouldn't we do something like this?
```
check_func = stdnum.util.get_cc_module('il', 'hp').is_valid if self.is_company else stdnum.util.get_cc_module('il', 'hp').is_valid
```
Since company and individual VATs are different, this would be more accurate. What do you think?

Reply:
@alialfie I think you are right. I read the difference between the two VATs elsewhere, and it said "incorporated businesses", I was not sure if it was the equivalent of companies... 
But it is pretty clear in the link you sent, I will modify this.